config.json

```diff
@@ -477,6 +477,17 @@
         "enums"
       ],
       "uuid": "b0a152e9-5a45-41f9-bda0-427111d9a56c"
+    },
+    {
+      "core": true,
```

Comment:
Can you make this non-core and make it unlocked by something topical, maybe `difference-of-squares`?

Reply:
I'll change that.